concordia/urls.py

```diff
@@ -63,7 +63,11 @@
     path("", views.HomeView.as_view(), name="homepage"),
     path("healthz", views.healthz, name="health-check"),
     path("about/", views.static_page, name="about"),
-    path("instructions/", views.static_page, name="instructions"),
+    path("help-center/", views.static_page, name="help-center"),
+    path("help-center/welcome-guide/", views.static_page, name="welcome-guide"),
```

Comment:
This reminds me: we should update the static page view to automatically generate breadcrumbs. Since it will work currently, I'm inclined to say that the way we do that is by having it just take the base portion of the filename, converting hyphens to spaces, and title-casing it (e.g. "help-center" ➡️ “Help Center”) but if we need to it can recursively load the parent pages all the way up and extract the title value.

Reply:
Yea I think this is a good idea. I can add it to the help center ticket